backend/git_real/activity_log_creators.py

```diff
@@ -11,12 +11,21 @@
 
 def log_pr_merged(pull_request):
     event_type, _ = EventType.objects.get_or_create(name=PR_MERGED)
-    LogEntry.objects.get_or_create(
-        actor_user=None,  # we don't care who clicked the merge button. Maybe we'll fill this in later
+    match = LogEntry.objects.filter(
+        actor_user=None,
```

Comment:
I think it would atually be useful to know who clicked the merge button. It'll give us more info about people's behaviour